oe_theme.theme

```diff
@@ -1243,3 +1243,56 @@ function oe_theme_preprocess_pattern_link__preview(array &$variables): void {
   // Pattern preview will pass along a \Drupal\Core\Render\Markup object.
   $variables['url'] = Url::fromUserInput((string) $variables['url']);
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ *
+ * Trim the description field on Featured items.
```

Comment:
This preprocessor can do multiple things. Among them is the trimming. So the comment here should be either just the first line or if we wanna be more descriptive we can say it preprocesses the featured item pattern. But you can leave it out since the rest don't seem to have it either. Same for the list item.

Reply:
You are right, I keep it consistent now and add a small comment inside the method.